dev/WindowsAppSDK_Insights/WindowsAppSDKInsightsLib/WindowsAppSDKInsightsLib.vcxproj

```diff
@@ -149,6 +149,7 @@
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>_DEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
+      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
```

Comment:
>MultiThreadedDebug [](http://example.com/codeflow?start=22&length=18)

Why is this added only for Debug/x64 and not for other configurations/platform?

Reply:
Good catch! I'll fix that.

Reply:
Done